tests/output_failures/wavefunction_basis_name.json

```diff
@@ -0,0 +1,59 @@
+{
+  "schema_name": "qc_schema_output",
+  "schema_version": 1,
+  "molecule": {
+    "schema_name": "qcschema_molecule",
+    "schema_version": 2,
+    "geometry": [
+      0.0,
+      0.0,
+      -0.1294769411935893,
+      0.0,
+      -1.494187339479985,
+      1.0274465079245698,
+      0.0,
+      1.494187339479985,
+      1.0274465079245698
+    ],
+    "symbols": [
+      "O",
+      "H",
+      "H"
+    ]
+  },
+  "driver": "energy",
+  "model": {
+    "method": "B3LYP",
+    "basis": "cc-pVDZ"
+  },
+  "keywords": {},
+  "provenance": {
+    "creator": "QM Program",
+    "version": "1.1",
+    "routine": "module.json.run_json"
+  },
+  "return_result": -76.4187620271478,
+  "success": true,
+  "properties": {
+    "calcinfo_nbasis": 24,
+    "calcinfo_nmo": 24,
+    "calcinfo_nalpha": 5,
+    "calcinfo_nbeta": 5,
+    "calcinfo_natom": 3,
+    "return_energy": -76.4187620271478,
+    "scf_one_electron_energy": -122.5182981454265,
+    "scf_two_electron_energy": 44.844942513688004,
+    "nuclear_repulsion_energy": 8.80146205625184,
+    "scf_dipole_moment": [
+      0.0,
+      0.0,
+      1.925357619589245
+    ],
+    "scf_iterations": 6,
+    "scf_total_energy": -76.4187620271478,
+    "scf_xc_energy": -7.546868451661161
+  },
+  "wavefunction": {
+    "basis": "cc-pVDZ"
```

Comment:
Unsure if we should allow this kind of specification, it creates a lot of ambiguity and requires downstream programs to tote around full basis libraries which may be incomplete.

Reply:
I agree. This is a test of an invalid output, to emphasize that basis must be fully specfied; it cannot be just the name of the basis set.